docs/spec.md

```diff
@@ -2027,6 +2027,16 @@ If `batch_group_count > 1`:
 * `result = concatenate(results, output_feature_dimension)`.
 <!-- markdownlint-enable line-length -->
 
+For quantized types, performs `dequantize_op_quantize(
+    lambda lhs, rhs: convolution(lhs, rhs, window_strides, padding,
+        lhs_dilation, rhs_dilation, window_reversal, input_batch_dimension,
+        input_feature_dimension, input_spatial_dimensions,
+        kernel_input_feature_dimension, kernel_output_feature_dimension,
+        kernel_spatial_dimensions, output_batch_dimension,
+        output_feature_dimension, output_spatial_dimensions,
+        feature_group_count, batch_group_count, precision_config), lhs, rhs,
+    type(result))`.
+
```

Comment:
>> If it is dequantize_op_quantize, are C29 and C31 of convolution still relevant?
Yes, there are relevant as the constraints are on the syntax of the quantized convolution. The semantics is modified to first dequantize and then do the operations in floating-point domain. Please let me know if this makes sense. 

>> Also, I think allowing hybrid quantization(relaxing C28?) now makes more sense with this change

Switching the spec to hybrid op does not change the semantics much: Instead of dequantizing both lhs and rhs, we need to to dequantize just the rhs.